fili-sql/src/main/java/com/yahoo/bard/webservice/sql/presto/PrestoSqlBackedClient.java

```diff
@@ -246,13 +246,7 @@ private static String fetchToLimitHelper(String fixQuotePrestoQuery) {
                     rows += (nextPart.charAt(i) - '0');
                 }
             }
-            int idxOfLineReturn = nextPart.indexOf('\n');
-            if (idxOfLineReturn != -1) {
-                nextPart = "LIMIT " + rows + nextPart.substring(idxOfLineReturn);
-
-            } else {
-                nextPart = "LIMIT " + rows;
-            }
+            nextPart = nextPart.replace("FETCH NEXT " + rows + " ROWS ONLY", "LIMIT " + rows);
```

Comment:
Is there any way to provide unit test coverage for this change?

Reply:
Added a UT for this class and included *Test.java in maven-surefire-plugin to enable the test.